playground/Stress/Stress.AppHost/aspire-manifest.json

```diff
@@ -1,193 +1,195 @@
 {
+  "$schema": "https://json.schemastore.org/aspire-8.0.json",
   "resources": {
     "stress-apiservice": {
       "type": "project.v0",
       "path": "../Stress.ApiService/Stress.ApiService.csproj",
       "env": {
         "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EXCEPTION_LOG_ATTRIBUTES": "true",
         "OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EVENT_LOG_ATTRIBUTES": "true",
+        "OTEL_DOTNET_EXPERIMENTAL_OTLP_RETRY": "in_memory",
         "ASPNETCORE_FORWARDEDHEADERS_ENABLED": "true"
       },
       "bindings": {
         "http-5180": {
           "scheme": "http",
           "protocol": "tcp",
           "transport": "http",
-          "targetPort": 5180
+          "port": 5180
```

Comment:
This doesn't represent a break that happened, it was just an error the schema caught?

Reply:
This was just a result of manifest regen.